### DISCUSSION TITLE
Direct paste from tab and index

### DISCUSSION THREAD

**Discussion by nrpeyton** [hluk/CopyQ#2828] [General]
Hello,

I am trying (unsuccessfully) to paste directly from a **tab**  _"Notes"_.

Despite adding the new command below, copyq instead pastes from the **default** _"&clipboard"_ **tab**, not from my manually added _"Notes"_ **tab**.

```
copyq tab "Notes"
copyq select 1
copyq paste
```

What am I doing wrong?

Thanks,

Nick

**Comment by hluk** [comment]
You need to use `tab` and `select` in a single command, otherwise each new `copyq` invocation uses by default the first tab.

```
copyq tab Notes select 1
```
